### PR TITLE
Views

### PR DESCRIPTION
**Pull request by dazehinn**
Update views to match the wireframe design

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class UsersController < ApplicationController
+  def index; end
+
```
Comment (TadesseDev) [inline app/controllers/users_controller.rb]: - [ ] Dear @dazehinn, according to the requirement It's required the users_controller `#index` action renders all users to their corresponding Views index page `(app/views/users/index.html.erb)`, currently you have an empty `index action`. Please fetch all users here.

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+  def show; end
```
Comment (TadesseDev) [inline app/controllers/users_controller.rb]: - [ ] Also in the `app/views/users/show.html.erb` I notice you have used `user.last_posts.each` but in the corresponding controller action `users_controller` of `#show` action you have not declared this instance, Please initialize the instance before using it.

File: app/views/users/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+          <li class="card">
+            <div class="flex">
+              <div class="photo-container"></div>
+              <div class="user-information flex">
+                <h2><%= user.name %></h2>
+                <p>Number of posts: <%= user.posts_counter %></p>
+              </div>
+            </div>
+          </li>
+        <% end %>
```
Comment (TadesseDev) [inline app/views/users/index.html.erb:15]: - [ ] Dear @dazehinn, You did well displaying the `user.name` and `user.posts_counter` of the user, But as the requirement mentioned we need to display the `user.photo` alongside these details. Please add a display for the user photo.
- [Optional] Consider using partials to render the user. create a file in `app/views/users/_user.html.erb` and add the details for displaying a single user in there, once that is done you can reuse this partial for displaying the user details.

File: app/views/posts/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,42 @@
+<main>
+  <section class="card-container">
+    <div class="card flex"> 
+      <div class="photo-container"></div>
+      <div class="user-information flex">
+        <h1><%= @user.name %></h1>
+        <p>Number of posts: <%= @user.posts_counter %></p>
+      </div>
+    </div>
+  </section>
```
Comment (TadesseDev) [inline app/views/posts/index.html.erb:10]: - [ ] Dear @dazehinn, It's my belief there was a manual change to the migration file, This caused an issue while I try to migrate the project and start it. As you can see there is a migration pending but could not migrate the project. `This prevents me from checking the project locally`. Please make sure its possible to drop and migrate your project database before the next review request. 
### The error on the console. 
image
### On the browser
image

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @dazehinn 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7




## Highlights
 - [ ] All linter passed 👍 
## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
- [Optional] Please try to be more descriptive in your pull request sumery. This is a place where you can introduce what changes you have made since the last merge. You can also specify any information that you would like to communicate with the code reviewer of the project.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @dazehinn, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  Great job. Your app works as expected. :heavy_check_mark: 
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Gitflow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**